doc/source/reference/routines.array-creation.rst

```diff
@@ -66,7 +66,7 @@ Creating character arrays (:mod:`numpy.char`)
 ---------------------------------------------
 
 .. note:: :mod:`numpy.char` is the preferred alias for
-   :mod:`numpy.core.defchararray`.
+   ``numpy.core.defchararray`` module.
```

Comment:
This one however makes sense, since it's pointing out that `np.char` should be used instead of `np.core.defchararray` :+1:

Reply:
Thanks.